scripts/ci-e2e-compatibility-start-pipeline.py

```diff
@@ -20,15 +29,22 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 
 def parse_args():
+    target_slug = "mapbox/mapbox-sdk"
+
+    versions = [{'id':'default', 'name':'Default stable customer experience'},
+               {'id':'latest', 'name':'Latest releases'},
+               {'id':'trunk', 'name':'Latest commit of the default branch'}]
+    versions = dict({v['id'] : v['name']  for v in versions})
```

Comment:
why did you override `versions` ?

Reply:
So that they are in correct format for the argument parser below.